### DISCUSSION TITLE
how to calculate AUC

### DISCUSSION THREAD

**Discussion by Chickenlover0908** [nlmixrdevelopment/nlmixr#618] [Q&A]
Hi, nlmixr team. I am a new bird both in PPK and R, there are nobody know these in my university, so my questions are simple (or stupid), you guys are the only person who can solve my problems, thank you very much for your answer.
i have three questions:
1. After fitting model, how can I calculate PK parameters like AUC in R?
2. In one cpt model, what is oral dose “cmt” in row data?
3. In the data provide in nlmixr—“oral_1cpt”， “SS” for DV is 99, what does it mean?

thank you very much, your package really helped me a lot.

**Comment by mattfidler** [comment]
> After fitting model, how can I calculate PK parameters like AUC in R?

Possibly by using PKNCA (though there are other R-based solutions too); And if you are using ODEs you can calculate it in the model too; See:

https://github.com/nlmixr2/rxode2/discussions/243#discussioncomment-3415260

> In one cpt model, what is oral dose “cmt” in row data?

Typically `depot` in the solved system.  But it can be anything in an ODE

> In the data provide in nlmixr—“oral_1cpt”， “SS” for DV is 99, what does it mean?

Nothing.  I think it was a flag that @RikSchoemaker used for sub-setting the data

**Reply by Chickenlover0908** [reply]
Thank you very much for answering such a simple question in person. Your package is very helpful to me. Thank you for your contribution to population pharmacokinetics.
